### DISCUSSION TITLE
Create Sub Domain, Easy Engine.

### DISCUSSION THREAD

**Discussion by malik-jpg** [EasyEngine/easyengine#1683] [Support Requests]
Can some one help me, i want to create sub domain with ee. im using droplet DO. 
does creating a subdomain have to create a new droplet? or not.

 ee site create android.ojuselementary.com --wpsubdom --ssl=le --cache=on --proxy-cache=on
Starting site creation.
Configuring project.
Creating WordPress site android.ojuselementary.com
Copying configuration files.
Starting site's services.
Downloading and configuring WordPress.
Moved /var/www/htdocs/wp-config.php to /var/www/wp-config.php successfully
Checking and verifying site-up status. This may take some time.
..............................................................
Warning: Problem connecting to site!
Warning: Initiating clean-up.
[android.ojuselementary.com] Docker Containers removed.
[android.ojuselementary.com] site root removed.
Success: Site android.ojuselementary.com deleted.
Report bugs here: https://github.com/EasyEngine/site-type-wp

HELP ME

**Comment by uebmaster** [comment]
i think that yoo only need execute the command like any other wp site, you don't need the --wpsubdomain argument.

**Comment by patil-vipul** [comment]
Add **--wildcard** to create wildcard SSL on root domain to work with sub domain.

**Comment by marcoandrei** [comment]
@malik-jpg 

1. You don't need to create a new droplet.
2. You must create a new site with the (sub)domain you need.
3. There is no such option "wpsubdom". Check the documentation: https://easyengine.io/commands/site/create/
